_data/maintainers.yml

```diff
@@ -1041,6 +1041,10 @@ thoughtchad:
   label: Chad Lancour
   name: thoughtchad
   url: https://github.com/thoughtchad
+timeplus-io:
+  label: timeplus-io
```

Comment:
Suggest using this:
timeplus:
  label: Timeplus
  name: timeplus-io
  url: https://timeplus.com

Reply:
@jovezhong updated